main.py

```diff
@@ -186,6 +186,7 @@ async def compile_project(project_root: Path, manifest: PackageManifest):
             path.parent.mkdir(parents=True, exist_ok=True)
             path.write_text(source.fetch_content())
     project = Project(project_root)
+    project.load_contracts()
     try:
         compiled_manifest = project.extract_manifest()
         results[project_root.name] = compiled_manifest
```

Comment:
`VyperCompilerError` should probably be `CompilerError` now on 0.8:

https://github.com/ApeWorX/ape/blob/main/src/ape/managers/compilers.py#L160-L163
(
```python
if errors:
            formatted_errors = [f"{e}" for e in errors]
            error_message = "\n\n".join(formatted_errors)
            raise CompilerError(error_message)
```
)

because all compiler-api errors get wrapped up in the end into a single `CompilerError`